classes/requests/helpers/class-kp-order-data.php

```diff
@@ -242,31 +242,32 @@ public function get_klarna_customer_object( $customer_type = null ) {
 
 		$strip_postcode_spaces = apply_filters( 'wc_kp_remove_postcode_spaces', false );
 		$customer_data         = $this->order_data->customer;
+		$checkout              = WC()->checkout();
```

Comment:
I believe the `get_klarna_customer_object` method can be called for both carts and orders. So this might cause it to break in the pay for order flow. I think a slight refactor of this code might be good so we can implement a helper method to get the value of the fields. We can discuss that internally though.